### DISCUSSION TITLE
click error

### DISCUSSION THREAD

**Discussion by Art-Che** [AutomaApp/automa#1069] [Q&A] [answered]
Hi!
There is a task of selecting values from a selector.
First click on selector.
Second click on value.

On some selectors, Automa selects a value from another item. This happens when they are located in the same ordinal places.
How can I fix it?

First selector
**First click:** .side-panel-iframe |> .ui-entity-editor-content-block:nth-child(6) .main-ui-control
**Second click:** .side-panel-iframe |> div.main-ui-select-inner-item:nth-child(1)

Second selector
**First click:**  .side-panel-iframe |> .ui-entity-editor-content-block:nth-child(9) .main-ui-control
**Second click:** .side-panel-iframe |> .main-ui-select-inner-item:nth-child(1) > .main-ui-select-inner-label

![1](https://user-images.githubusercontent.com/120256667/211242270-cb20b3e2-5eb8-41c5-a487-6f2cd9ad4f51.png)
![2](https://user-images.githubusercontent.com/120256667/211242277-abba1d44-0c5a-439c-a097-61c931f51356.png)

**Comment by Kholid060** [comment, marked as answer]
That might happen because the first and second value selector references the same element. I guess you can fix it by manually writing the selector.

**Reply by Art-Che** [reply]
Thanks for the answer!
